Sources/Orders/OrdersError.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  OrdersError.swift
+//  PassKit
+//
+//  Created by Francesco Paolo Severino on 30/06/24.
+//
+
+public enum OrdersError: Error {
```

Comment:
Before you release the major version this should be hidden behind a struct as in https://github.com/vapor/jwt-kit/blob/main/Sources/JWTKit/JWTError.swift IMO as adding/removing cases to this (which is likely) is source breaking and requires a new major release